packages/cli/src/lib/utils.ts

```diff
@@ -76,18 +76,8 @@ export const getAxeSource = (axePath?: string): string | void => {
     return;
```

Comment:
I'm confused by what's happening here.

If the file `axePath` exists, return nothing. Otherwise, return the contents of a file.

Reply:
Refactored this ☝️